torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2170,14 +2171,40 @@ def call_function(
         )
 
 
-class AutoFunctionalizeHigherOrderVariable(TorchHigherOrderOperatorVariable):
+class PassthroughHigherOrderVariable(TorchHigherOrderOperatorVariable):
```

Comment:
@xmfan there's two ways to go around this:
1) We can create some sort of GenericHOPVariable that works for simple HOPs. 
2) We can create a FlexAttentionBackwardHOPVariable for just FlexAttentionBackward.

2 is a lot simpler to do. If you want the task of doing 1 because it is more generic, there are some things missing in this PR (that I'll comment on inline)